src/components/FeedSingleTweet.tsx

```diff
@@ -90,6 +91,12 @@ const FeedSingleTweet = ({
   // const handleReplyClick = () => {
   //   return <ReplyTweet />;
   // };
+  const urlSearch = new RegExp(
+    "/https?://(?:www.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9].[^s]{2,}|https?://(?:www.|(?!www))[a-zA-Z0-9]+.[^s]{2,}/"
+  );
+
+  const url = content.match(urlSearch) || "";
```

Comment:
```suggestion
  const url = content.match(urlSearch) || [];
```
to keep consistency with types